circleci/images/Makefile

```diff
@@ -11,6 +11,7 @@ endif
 
 # all postgres versions we test against
 PG_VERSIONS=13.4 14.0
+PG_SOURCE_VERSIONS=15~~devel+git+master
```

Comment:
This will break when PG15 is released and master is bumped to 16devel. Is this safe to assume master has PG15?

Reply:
Correct. However, currently there is no branch for pg15 yet. That will come at the same time we don't need to build pg15 from source anymore. By that time this build starts failing, because the dev build system will then start creating `16~~devel` packages.